### DISCUSSION TITLE
2011/03/22/%E7%94%A8%E4%B8%8A%E4%BA%863g%E4%B8%8A%E7%BD%91

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#320] [Announcements]
# 2011/03/22/%E7%94%A8%E4%B8%8A%E4%BA%863g%E4%B8%8A%E7%BD%91

前一阵子为了方便给豆豆喂奶，在老婆单位附近租了套房子。预计租期不会太久，也就不打算装宽带什么的了。可是上网已经成为我生活一部分了，离开它很多事情都办不了，于是决定搞个无线上网临时用用。

https://blog.qizhen.xyz/2011/03/22/%E7%94%A8%E4%B8%8A%E4%BA%863g%E4%B8%8A%E7%BD%91

**Comment by ruanqizhen** [comment]
之前博客下的留言：

陈小刚
2011年03月30日 16:40 编辑
3G相对来说还是联通的稍微好一点，起码速度比较快，但是价格也相对贵一点。

回复
勺子
2011年03月23日 12:29 编辑
勺以前也用过，就为在外面没有无线网的地方也可以上上网，结果那速度真的很让人蛋疼的，不过，了胜于无拉。。。

回复
勺子
2011年03月23日 12:30 编辑
并不像商家宣传的那样的，人家取的是上限峰值，不厚道的。

回复
